src/CategoryLookup.php

```diff
@@ -130,6 +132,15 @@ public function on_edit( $category_id ) {
 		$this->update( $category_id );
 	}
 
+	/**
+	 * When a product category gets created, add a new lookup data.
```

Comment:
```suggestion
	 * When a product category gets created, add a new lookup row.
```

Reply:
Thank you 👍 Fixed in https://github.com/woocommerce/woocommerce-admin/pull/7290/commits/ae90368f5ca2f7a1ad652042c00eeff353366036